### ISSUE TITLE
Bug: Dummy chapters creates 9 million chapters for a two-hour film

### ISSUE THREAD

**Issue by LordMike** [MediaBrowser/Emby#97]
Hi,

I get mutliple OOM exceptions when using FFprobe on "Cube 2" (5417 seconds).
The problem causing the OOM, is that the dummy chapters created goes into the range of 9 million+ chapters, so the JsonSerializer crashes when trying to serialize it in-memory.

The 9 million chapters come from a gigantic number for the length of the movie, which in turn comes from FFprobe. The number is _always_ written with a point for decimal seperator, while on my computer (Danish locale), it's parsed as a thousand-separator.

So 5417.556667 seconds becomes 5417556667 seconds.

**Bottom line:** Conversion in file:
.\MediaBrowser.Controller\Providers\MediaInfo\FFProbeVideoInfoProvider.cs 
Method: 
protected override Task Fetch(Video video, CancellationToken cancellationToken, FFProbeResult data, IIsoMount isoMount)

Line (# 197 here):
video.RunTimeTicks = TimeSpan.FromSeconds(double.Parse(data.format.duration)).Ticks;

.. Needs a culture. It is likely that other places that use data from FFProbe are affected by the issue.
